docs/team/markusyeo.md

```diff
@@ -9,35 +9,37 @@ Dengue Hotspot Tracker is a desktop app for managing dengue cases. The user inte
 
 Given below are my contributions to the project.
 
+* Given below are my contributions to the project.
+
+* **New Feature**: Changed default file storage format to be CSV
+  * Justification: As we deal with tabular data, we would like users to be able to easily view the data entered into `DengueHotspotTracker` offline.
+  Additionally NEA personnel are more likely to be familiar with CSV than json!
+  * Highlight: It was challenging to write to CSV as it required exploring a new package `OpenCSV` this was nonetheless a very fruitful experience in exploring a new codebase and adding it into our project. 
+  I had to add a new `CsvUtil` file to handle all the reading and writing from CSV.
+* **New Feature**: Added the ability to import and export data as a csv file
+  * Justification: This feature allows users to edit csv files to be able to import into `DengueHotspotTracker` and export from.
+  Additionally allowing users to export files, edit them offline, then import them back into the DHT, is more convenient for sharing information with those without direct access to the DengueHotspotTracker.
+  * This feature also pairs with the `find` function with allows users to filter the data and also export that specific set of data to be shared with others.
+* **New Feature**: Added the ability to checkout overview data as a csv file
```

Comment:
It might be good to explain what "checkout" means! Since it sounds just like "export" :O